core/test/unit/data-generator/batch-replayer.test.ts

```diff
@@ -138,4 +138,17 @@ test("BatchReplayer should create a step function", () => {
   template.resourceCountIs("AWS::StepFunctions::StateMachine", 1);
 });
 
+test("BatchReplayer should not have additionalStepFunctionTasks by default", ()=> {
+  expect(batchReplayer.additionalStepFunctionTasks).toBeUndefined();
+});
+
+test("BatchReplayer should allow additionalStepFunctionTasks as empty list", ()=> {
```

Comment:
can we add a test to check the tasks are added to the CFN template?

Reply:
Yes.